migration/20221014150625-modify-student-table.js

```diff
@@ -0,0 +1,13 @@
+const { MigrationInterface, QueryRunner } = require("typeorm");
+
+module.exports = class modifyStudentTable20221014150625 {
+
+    async up(queryRunner) {
```

Comment:
You should the down method also with a script to remove the added columns.
@RamaMahjoub